lang/de/lang.php

```diff
@@ -13,3 +13,13 @@
 $lang['tin'] = 'temporär angemeldet';
 $lang['fail'] = 'gescheiterter Anmeldeversuch';
 
+$lang['data']   = 'Zusatzdaten';
+$lang['submit'] = 'Filtern';
+$lang['filter_auth_ok'] = 'An- und Abmeldungen';
+$lang['filter_auth_error'] = 'Fehlgeschlagene Anmeldeversuche';
+$lang['filter_admin'] = 'Admin';
+$lang['filter_other'] = 'Andere';
+
+$lang['email_max_success_logins_subject'] = 'Warnung: Zu viele Anmeldungen auf';
+$lang['email_max_failed_logins_subject'] = 'Warnung: Zu viele fehlgeschlagene Anmeldeveruche auf';
```

Comment:
It seems you're adding something to a sentence here? Because word order is different in different languages, a parameter should be use instead. Though I would simply leave it out here, the default mail prefix should make it clear where the mail originates from.

Reply:
Good point, I will leave the domain out of the subject line.